app/lib/azul_search_service.rb

```diff
@@ -236,39 +236,29 @@ def self.is_analysis_file(file_summary)
   def self.get_file_summary_info(accessions)
     client = ApplicationController.hca_azul_client
     file_query = { 'project' => { 'is' => accessions } }
-    if client.query_too_large?(file_query)
-      # cut query in half to prevent HTTP 413
-      queries = accessions.each_slice((accessions.size / 2.0).round).map { |list| { 'project' => { 'is' => list } } }
-    else
-      queries = [{ 'project' => { 'is' => accessions } }]
-    end
-    all_results = []
-    Parallel.map(queries, in_threads: queries.size) do |query|
-      results = client.projects(query: query)
-      results['hits'].map do |entry|
-        entry_hash = entry.with_indifferent_access
-        project_hash = entry_hash[:projects].first # there will only ever be one project here
-        accession = project_hash[:projectShortname]
-        result = {
-          study_source: 'HCA',
-          name: project_hash[:projectTitle],
-          accession: accession,
-          description: project_hash[:projectDescription],
-          studyFiles: [
-            {
-              project_id: project_hash[:projectId],
-              file_type: 'Project Manifest',
-              count: 1,
-              upload_file_size: 1.megabyte, # placeholder filesize as we don't know until manifest is downloaded
-              name: "#{accession}.tsv"
-            }
-          ]
-        }.with_indifferent_access
-        project_file_info = extract_file_information(entry_hash)
-        result[:studyFiles] += project_file_info if project_file_info.any?
-        all_results << result
-      end
+    results = client.projects(query: file_query)
```

Comment:
Could you confirm this logs to Sentry and Mixpanel if HCA Azul responds problematically?

In my preliminary diagnosis, I couldn't find such logs.  (Given its unique querying and reporting functionality, having such error logs in Mixpanel helps, in addition to Sentry.)

Reply:
As far as I know, yes - though I'm puzzled as to why we didn't see them before.  It's possible it had something to do with the parallelization of the requests, though I doubt it (we see them for the `cURL` configs).

I manually inserted an encoding error into this method and re-ran the tests, then configured Sentry locally and was able to get [this](https://broad-institute.sentry.io/issues/5137727763/) error to show up, so I'm confident we should see these moving forward.